styles/Canonical/004-Canonical-product-names.yml

```diff
@@ -0,0 +1,37 @@
+extends: substitution
+message: "Did you really mean'%s' instead of '%s'?"
```

Comment:
1. I know this is Vale's own example for a substitution message, but I'm partial to something less mysterious and more straightforward like `"Use '%s' instead of '%s'"`.

2. Small typo: space missing in `mean'%s'`

Approving anyway, since these are nitpicks.